devicetypes/smartthings/smartsense-temp-humidity-sensor.src/smartsense-temp-humidity-sensor.groovy

```diff
@@ -214,3 +230,11 @@ def configure() {
 private Boolean isFrientSensor() {
 	device.getDataValue("manufacturer") == "frient A/S"
 }
+
+private boolean isEWeLink() {
+	device.getDataValue("manufacturer") == "eWeLink"
+}
+
+private boolean isEWeLinkTh01() {
+	isEWeLink() && device.getDataValue("model") == "TH01"
+}
```

Comment:
could we get rid of the separate `isWeLink` function?

Reply:
hi~ I have modified the file.